Shared/Samples/Show utility associations/ShowUtilityAssociationsView.swift

```diff
@@ -0,0 +1,268 @@
+            map.addUtilityNetwork(network)
+        }
+        
+        deinit {
+            ArcGISEnvironment.authenticationManager.arcGISCredentialStore.removeAll()
+        }
+        
+        // MARK: Methods
+        
+        /// Performs important tasks including adding credentials, loading and adding operational layers.
+        func setup() async throws {
+            try await ArcGISEnvironment.authenticationManager.arcGISCredentialStore.add(.publicSample)
+            try await network.load()
+            addLayers()
+        }
+        
+        private func addLayers() {
+            // Get all the edges and junctions in the network.
+            guard let networkSources = network.definition?.networkSources else { return }
+            let sourcesByType = Dictionary(grouping: networkSources, by: \.kind)
+            
+            // Add all edges that are not subnet lines to the map.
+            let edgeLayers = sourcesByType[.edge, default: []]
+                .filter { $0.usageKind != .subnetLine }
+                .map { FeatureLayer(featureTable: $0.featureTable) }
+            
+            map.addOperationalLayers(edgeLayers)
+            
+            // Add all the junctions to the map.
+            let junctionLayers = sourcesByType[.junction, default: []]
+                .map { FeatureLayer(featureTable: $0.featureTable) }
+            map.addOperationalLayers(junctionLayers)
+        }
+        
+        static func makeAssociationsOverlay() -> GraphicsOverlay {
+            let attachmentValue = UniqueValue(
+                description: "Attachment",
+                label: "",
+                symbol: .attachment,
+                values: [UtilityAssociation.Kind.attachment]
+            )
+            let connectivityValue = UniqueValue(
+                description: "Connectivity",
+                label: "",
+                symbol: .connectivity,
+                values: [UtilityAssociation.Kind.connectivity]
+            )
+            
+            let renderer = UniqueValueRenderer(
+                fieldNames: ["AssociationType"],
+                uniqueValues: [attachmentValue, connectivityValue],
+                defaultLabel: ""
+            )
+            
+            let overlay = GraphicsOverlay()
+            overlay.renderer = renderer
+            return overlay
+        }
+        
+        func addAssociationGraphics(viewpoint: Viewpoint, scale: Double) async throws {
+            // Check if the current viewpoint is outside of the max scale.
+            guard isAuthenticated, scale <= maxScale, !isAddingGraphics else { return }
+            isAddingGraphics = true
+            let extent = viewpoint.targetGeometry.extent
+            // Get all of the associations in extent of the viewpoint.
+            let associations = try await network.associations(forExtent: extent)
+            let existingAssociationIDs = Set(associationsOverlay.graphics.map { $0.attributes["GlobalId"] as? UUID })
```

Comment:
The type of `existingAssociationIDs` is `[UUID?]`:

![image](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/2257493/6fd4dc0d-e673-474d-bf97-d2adf552d2bd)

But it doesn't need to be. It would be better if it was just `[UUID]`.

```suggestion
            let existingAssociationIDs = associationsOverlay.graphics
                .compactMap { $0.attributes["GlobalId"] as? UUID }
```

![image](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/2257493/0b85eb18-1686-4c7b-a5ec-981629c07696)